pipeline/build-push-gke-deploy/0.1/README.md

```diff
@@ -18,7 +18,7 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/
 ## Install the Pipeline
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/pipeline/build-push-gke-deploy/0.1/build-push-gke-deploy.yaml
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/pipeline/build-push-gke-deploy/0.1/build-push-gke-deploy.yaml
```

Comment:
the following lines also needs to be updated :stuck_out_tongue: 
https://github.com/tektoncd/catalog/pull/672/files#diff-ce5e093d42f4b89b783f2120f3d45377fe577047eacd4516ba3c2ff224830799L11-L16